src/css/base.scss

```diff
@@ -3457,7 +3457,7 @@ textarea,
 }
 
 .dc__flip-90 {
-    transform: rotate(-90deg);
+    transform: rotate(90deg);
```

Comment:
is it intentional? It has been used on multiple places

Reply:
Yes, there were duplicate styles of 270 deg and -90deg, have used 270 deg style wherever the -90 deg was needed. And created this 90 deg rotation for an icon used in `Deployment status` card